test/e2e/e2e.go

```diff
@@ -472,6 +472,14 @@ func deleteClusterExternalContainer(containerName string) {
 	}, 5).Should(gomega.HaveLen(0))
 }
 
+func getClusterExternalContainerIP(containerName string) string {
```

Comment:
We don't need this function the `createClusterExternalContainer` returns ipv4 and ipv6 from the external container, just store those and use them.

Reply:
ACK, thanks